tests/test_phot_util.py

```diff
@@ -98,21 +96,23 @@ def test_computeWidths(self):
     def test_getRandomDiffRmsInMmags(self):
         """Test random sampling of magnitude diffs."""
         # Ensure measurements are deterministic
-        random.seed(8675309)
+        seed = 8675309
+        rng = np.random.default_rng(seed)
 
-        expected = 848.5281374238564
+        expected = -1237.436867076458
```

Comment:
I'm disturbed that what you get when you call 867-5309 is an RMS diff of >1 mag, but if that's what comes out, then OK.

Reply:
This is an example of a test where we can create synthetic data. The input is the following array, `mag = np.linspace(20, 25, 101)`, i.e., with intentionally large distribution for testing purposes.